incremental-font-transfer/src/bin/ift_extend.rs

```diff
@@ -84,3 +107,102 @@ fn main() {
     std::fs::write(&args.output, font_bytes).expect("Writing output font failed.");
     println!(">> Wrote patched font to {}", &args.output.display());
 }
+
+fn parse_unicodes(args: Vec<String>, codepoints: &mut IntSet<u32>) -> Result<(), ParsingError> {
+    for unicode_string in args {
+        if unicode_string.is_empty() {
+            continue;
+        }
+
+        if unicode_string == "*" {
+            let all = IntSet::<u32>::all();
+            codepoints.union(&all);
+            return Ok(());
+        }
+        let Ok(unicode) = unicode_string.parse() else {
+            return Err(ParsingError::UnicodeCodepointParsingFailed(unicode_string));
+        };
+        codepoints.insert(unicode);
+    }
+    Ok(())
+}
+
+fn parse_fixed(value: &str, flag_value: &str) -> Result<Fixed, ParsingError> {
+    f64::from_str(value)
+        .map_err(|_| ParsingError::DesignSpaceParsingFailed {
+            flag_value: flag_value.to_string(),
+            message: "Bad axis position value".to_string(),
+        })
+        .map(Fixed::from_f64)
+}
+
+fn parse_design_space(args: Vec<String>) -> Result<DesignSpace, ParsingError> {
+    let re = Regex::new(r"^([a-zA-Z][a-zA-Z0-9 ]{3})@([0-9.]+)(:[0-9.]+)?$").unwrap();
```

Comment:
I wouldn't personally use a regex for this, the parsing seems straightforward if you just split at `@`? regex is fairly heavyweight.

Reply:
Agreed it's typically heavy weight; however, in this case performance isn't much of a concern as this is a command line utility for testing/debugging.